### DISCUSSION TITLE
Where is the documentation for the config file settings?

### DISCUSSION THREAD

**Discussion by Jacob-Stevens-Haas** [microsoft/vscode-jupyter#14334] [Questions and Answers]
Was looking for how to override the warnings filters that Jupyter adds when running python kernels (specifically, it looks like  notebooks default to ignoring `UserWarning`).  I assumed it would be something similar to `python.terminal.launchArgs` within the `jupyter...` config (have a faint memory of finding it there once before).  I don't see any config documentation linked on [extension readme](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter&ssr=false) nor on [the extensions docs](https://code.visualstudio.com/docs/datascience/jupyter-notebooks).  Yet [this comment](https://github.com/microsoft/vscode-jupyter/issues/9351#issuecomment-1067117412)  from a dev in this repo indicates there's some docs with examples of config arguments.

I may be blind or just not searching for the correct terms, so sorry if this is a poor question.  But everything I google gives me  "How do I ignore warnings in notebooks?" -> "try using warnings.filterwarnings", which obviously isn't what I'm looking for.

**Comment by DonJayamanne** [comment]
The Jupyter extension does not start Jupyter Notebook/Lab server, hence there's no concept of supporting configuration files.

> o override the warnings filters that Jupyter adds when running python kernels (specifically, it looks like notebooks default to ignoring UserWarning). I assumed it would be something similar to python.terminal.launchArgs

Which warnings are you referring to?
Also, please can you share a screenshot of VS Code where these messages are displayed?

**Reply by Jacob-Stevens-Haas** [reply]
Sorry if I chose poor wording - I've actually solved the issue.

> there's no concept of supporting configuration files

I meant the "Settings" in VSCode/settings.json

> Which warnings are you referring to?

I was referring to the warnings filter in python.  I had thought ipython/jupyter was was adding things to python's warnings filter at startup.  It was actually a 3rd/party package import

Thanks for your response!
